stylesheets/colors.css

```diff
@@ -58,5 +58,15 @@
     --res-img-section: #e8deda;
     --sq-img-section: #e6d3cd;
     --rd-img-section: #ddd8a9;
+
+    --textbox-section: #f1f5f9;
+    --lbl-input-section: #eff3f3;
+    --val-input-section: #f5f5f5;
+    --radio-section: #eaf1ef;
+    --checkbox-section: #e9f0e8;
+
+    --textbox-bg: #f5f5f5;
+    --success-input: #008000;
+    --warn-input: #92680b;
```

Comment:
if you are using colors , use variable names like --COLOR-RED , capital and using COLOR in var it will populate a list when you code a color next in VSCODE.